net/wireguard/src/etc/rc.d/opnsense-wireguard

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+# PROVIDE: opnsense-wireguard
+# REQUIRE: SERVERS
+# KEYWORD: shutdown
+#
+
+. /etc/rc.subr
+
+name=wireguard
+
+stop_cmd=wireguard_stop
+start_cmd=wireguard_start
+status_cmd=wireguard_status
+rcvar=wireguard_enable
+
+load_rc_config opnsense-wireguard
+pidfile=`pgrep -f wireguard-go`
+command=/usr/local/bin/wg-quick
+
+[ -z "$wireguard_enable" ] && wireguard_enable="NO"
+
+# stop wireguard
+wireguard_stop()
+{
+    echo "stopping wireguard"
+    config=`cd /usr/local/etc/wireguard/ && ls -1 *.conf | cut -d. -f1`
```

Comment:
find(1) is better suited for this than cd && ls

Reply:
@fichtner find will usually add more information (path)

Reply:
+1 for `find` and please stop using the backquotes for subshells.